cdc/sink/mq/manager/kafka_manager.go

```diff
@@ -137,13 +228,44 @@ func (m *kafkaTopicManager) CreateTopic(topicName string) (int32, error) {
 	}
 	m.lastMetadataRefresh.Store(time.Now().Unix())
 
+	return nil
+}
+
+// CreateTopic creates a topic with the given name
+// and returns the number of partitions.
+func (m *kafkaTopicManager) CreateTopic(topicName string) (int32, error) {
```

Comment:
return the partition number seems useful

How about return metadata of the created topic directly?

Reply:
> return the partition number seems useful
> 
> How about return metadata of the created topic directly?

Currently, returning the partition number of a topic is enough for us. But returning `sarama.TopicMetadata` is also feasible. @hi-rustin What do you think?